src/openforms/validations/validators/formats.py

```diff
@@ -48,19 +49,18 @@ def __call__(self, value, submission):
                 )
 
 
-@register(
-    "phonenumber-international",
-    verbose_name=_("International phone number"),
-    for_components=("phoneNumber",),
-)
-class InternationalPhoneNumberValidator(PhoneNumberBaseValidator):
+@register("phonenumber-international")
+class InternationalPhoneNumberValidator(PhoneNumberBaseValidator, BasePlugin[str]):
```

Comment:
Probably this should also be renamed to `PhoneNumberValidatorMixin` similar to the `KVKRemoveValidatorMixin`?

The ABC base class conflicts a bit with the idea of `BasePlugin[str]` being a base too - which one dictates/enforces what? Using a `*Mixins, Base` inheritance order leans more towards composition than weird MRO challenges with multiple "true" bases.